launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -54,6 +54,10 @@ public class SparkLauncher extends AbstractLauncher<SparkLauncher> {
 
   /** Configuration key for the driver memory. */
   public static final String DRIVER_MEMORY = "spark.driver.memory";
+  /** Configuration key for the driver default extra class path. */
+  public static final String DRIVER_DEFAULT_EXTRA_CLASS_PATH =
+    "spark.driver.defaultExtraClassPath";
+  public static final String DRIVER_DEFAULT_EXTRA_CLASS_PATH_VALUE = "hive-jackson/*";
```

Comment:
So even users who use no hive distributions. they will also have this default class patch value in the extra class path, right?

Reply:
Yes, the configuration is simply pointing non-existing directly. As you can see in `make-distribution.sh` and PR description. `hive-jackson` directory is created only when there exist `jackson-*-asl-*.jar`s.